apps/reply/interface.html

```diff
@@ -0,0 +1,125 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="../../css/spectre.min.css">
+    <link rel="stylesheet" href="../../css/spectre-icons.min.css">
+</head>
+
+<body>
+    <form id="replyForm">
+        <label class="label">New Custom Reply:</label>
+        <input class="form-input" id="msg" type="text" required></input>
+        <input class="btn btn-primary" type="submit" value="Add">
+        <br>
+        <br>
+    </form>
+    <button class="btn btn-primary" onclick="updateDevice()">Update Device</button>
+    <div>
+        <div id="loading">
+            <div class="empty">
+                <div class="empty-icon">
+                    <div class="loading loading-lg"></div>
+                </div>
+                <p class="empty-title h5">Loading</p>
+                <p class="empty-subtitle">Syncing custom replies with your watch</p>
+            </div>
+        </div>
+        <div id="empty" class="d-hide">
+            <div class="empty">
+                <div class="empty-icon">
+                    <i class="icon icon-more-horiz"></i>
+                </div>
+                <p class="empty-title h5">No custom replies</p>
+                <p class="empty-subtitle">Use the field above to add a custom reply</p>
+            </div>
+        </div>
+
+        <table id="replyList" class="table table-striped table-hover"></table>
+    </div>
+
+    <script src="../../core/lib/customize.js"></script>
+    <script>
+        const form = document.querySelector("#replyForm");
+        form.addEventListener("submit", (event) => {
+            event.preventDefault();
+            saveReply();
+        });
+        var replies = [];
+        var fetching = true; // Bit of a hack
+        var el = document.getElementById('items');
+        function onInit(device) {
+            fetching = true;
+            Util.readStorageJSON("replies.json", (arr) => {
+                if (arr) { 
+                    replies = replies.concat(arr);
+                    fetching = false;
+                    renderReplyList();
+                }
+            });
+        }
+
+        function toggleVisibility() {
+            let empty = document.getElementById("empty");
+            let loading = document.getElementById("loading");
+            if (replies.length == 0) {
+                empty.setAttribute("class", "d-block");
+                loading.setAttribute("class", "d-hide");
+            }
+            else if (fetching) {

```

Comment:
Should these be the other way round? Testing `fetching`, then `replies.length` ?